Features/Lib/PlatformUI.js

```diff
@@ -0,0 +1,422 @@
+/*
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ * @version //autogentag//
+ */
+
+/**
+ * Container for Javascript functions
+ *
+ * @returns {PlatformUIHelper}
+ */
+function PlatformUIHelper()
+{
+    //YUI global object with node-event-simulate module
+    var Y = YUI().use('node-event-simulate', 'stylesheet');
+
+    /**
+     * Clicks on a specific HTML element
+     *
+     * @param {HTML ELement / string} element/selector
+     * @returns {Boolean}
+     */
+    this.clickElement = function (element)
+    {
+        // if using a selector, convert to Y element first
+        if ( typeof( element ) == 'string' )
+            element = Y.one( element );
+
+        if (!element)
+            return false;
+        element.simulate("focus");
+        element.simulate("mousedown");
+        element.simulate("mouseup");
+        element.simulate("click");
+        return true;
+    }
+
+    /**
+     * Mouse over on a specific HTML element
+     *
+     * @param {HTML ELement / string} element/selector
+     * @returns {Boolean}
+     */
+    this.mouseOverElement = function (element)
+    {
+        // if using a selector, convert to Y element first
+        if ( typeof( element ) == 'string' )
+            element = Y.one( element );
+
+        if (!element)
+            return false;
+        element.simulate("focus");
+        element.simulate("mouseover");
+        return true;
+    }
+
+    /**
+     * Get an HTML element using a selector and its text
+     *
+     * @param {string} text
+     * @param {sting} selector
+     */
+    this.getElementByText = function (text, selector)
+    {
+        var rightElement = null;
+        Y.all(selector).some(function (element) {
+            var textContent = Y.DOM.getText( element.getDOMNode() ).trim();
+            var textEquals = (textContent == text);
+            if ( textEquals )
+            {
+                rightElement = element;
+                return true;
+            }
+        }, this);
+
+        return rightElement;
+    }
+
+    /**
+     * Finds an element by text and clicks on it
+     *
+     * @param {string} text
+     * @param {string} selector
+     */
+    this.clickElementByText = function(text, selector)
+    {
+        this.clickElement(
+                this.getElementByText(text, selector)
+                );
```

Comment:
on the CS thing, the whole file is wrong, the CS for JavaScript are different. For instance, this method should be written like:
```js
this.clickElementByText = function(text, selector) {
    this.clickElement(this.getElementByText(text, selector));
};
```
I guess there's enough JS example in this bundle to get it :-)

But, I'm a bit surprise to see such file here, shouldn't the BDD framework provide such helpers ?